packages/components/src/styles/components/side-nav/main.scss

```diff
@@ -7,7 +7,20 @@
 // SIDE-NAV
 //
 
+:root {
+  // TODO: Move to sidenav tokens
+  --hds-app-sidenav-headerless-width-minimized: 16px;
+
+  // TODO: Create app-header token:
+  --hds-app-header-height: 59px;
```

Comment:
@didoo Should I create design tokens for these? I assume yes but want to confirm. (Could make them sass variables instead?)